src/views/Activate/MasterKeyConfirm.tsx

```diff
@@ -161,7 +169,13 @@ const MasterKeyConfirm = () => {
                             fontFamily={'Source Code Pro'}
                             borderRadius={'5px'}
                             border={'solid 1px rgba(0,0,0,0.25)'}
+                            borderColor={
+                              showError ? '#FF4136' : 'rgba(0,0,0,0.25)'
```

Comment:
can you name these colors as constants?

Reply:
shouldn't these colors be values from Indigo?

Reply:
yes, if Indigo exports colors as JS strings. I don't know enough about Indigo.